src/lib/calculator.js

```diff
@@ -0,0 +1,9 @@
+let calculator;
```

Comment:
this looks fine to me - i'll leave it to Chris if he thinks anything needs changing here.